client/flutter/lib/constants.dart

```diff
@@ -1,29 +1,35 @@
 import 'package:flutter/cupertino.dart';
 
-class Constants {
-  static const Color backgroundColor = CupertinoColors.white; // WHITE
-  static const Color greyBackgroundColor = Color(0xFFF9F8F7); // GREY
-  static const Color primaryColor = Color(0xff008DC9); // WHO BLUE
-  static const Color textColor = Color(0xff3C4245); //GREY
-  static const Color primaryDarkColor = Color(0xff1A458E); // NAVY
-  static const Color accentColor = Color(0xffD86422); // ORANGE
-  static const Color whoBackgroundBlueColor = Color(0xff007EB4); // WHO BLUE 2
-  static const Color whoAccentYellowColor = Color(0xffFFCC00); // YELLOW
-  static const Color bodyTextColor = Color(0xff272626); // CHARCOAL
-
-  static const Color emergencyRedColor = Color(0xffD82037);
-  static const double buttonTextSpacing = -.4;
-  static const Color menuButtonColor = Color(0xCC1694BE);
+class AppTheme {
+  final BuildContext context;
+  AppTheme(this.context);
+  Color get backgroundColor =>
+      isLight(context) ? CupertinoColors.white : CupertinoColors.black;
```

Comment:
What I think is AppTheme is just a class with uninitialized values for thsese colos.

Then you have two instances DarkTheme and LightTheme with the right values in each theme @theswerd